CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
-Tantivy 0.6
+Tantivy 0.???
 ==========================
 
+- Added support for parsing AllQuery and RangeQuery via QueryParser
+    - AllQuery: `*`
+    - RangeQuery:
+        - Inclusive `field:[startIncl to endIncl]`
```

Comment:
I think most of this has been released in tantivy0.6 actually but well...

Reply:
Is this true? It didn't look like it when I checked.

Reply:
I think you are right. Perfect.